test/Microsoft.ML.Tests/Scenarios/Api/AspirationalExamples.cs

```diff
@@ -60,5 +60,96 @@ public void FirstExperienceWithML()
                 PetalLength = 5.1f
             });
         }
+
+
+        [Fact]
+        public void SimpleIrisDescisionTrees()
+        {
+            var env = new TlcEnvironment(new SysRandom(0), verbose: true);
+            string dataPath = "iris-data.txt";
+            // Create reader with specific schema.
+            var dataReader = TextLoader.CreateReader(env, ctx => (
+               Label: ctx.LoadText(0),
```

Comment:
>L [](start = 15, length = 1)

Tom is using lowercase for those, in his StaticPipeTest. Shall we adhere to that?

Reply:
I guess I'd probably prefer the way it is, but let's see what Tom says

---
In reply to: [215033893](https://github.com/dotnet/machinelearning/pull/809#discussion_r215033893) [](ancestors = 215033893)

Reply:
Ok, let's rename to camelCase

---
In reply to: [215054683](https://github.com/dotnet/machinelearning/pull/809#discussion_r215054683) [](ancestors = 215054683)